lib/spotlight/dor/indexer.rb

```diff
@@ -48,47 +47,52 @@ def solr_document resource
     def resource druid
       Harvestdor::Indexer::Resource.new harvestdor, druid
     end
-    
+
     private
     # def add_something_to_solr_doc sdb, solr_doc
-    #   solr_doc['some_field_name_tsi'] = 'some value'
+    #   solr_doc['some_field_name_tsi'] = 'some value'   # OR
+    #   insert_field solr_doc,'some_multivalued_field_name_ssim',array_of_values
     # end
 
+    def add_donor_tags sdb,solr_doc
```

Comment:
similar to my github comment at sul-dlss/sul-exhibits-template/pull/56

I might like a comment in here or somewhere to indicate
1. it only applies to Feigenbaum as of 2015/10;  other collections won't have noise because of the way it's indexed
2. in the future we may refactor so that single exhibit specific configuration is possible